lib/utils/screen-manager.js

```diff
@@ -55,14 +44,14 @@ ScreenManager.prototype.render = function (content, bottomContent) {
   // setPrompt will change cursor position, now we can get correct value
   var cursorPos = this.rl._getCursorPos();
 
-  content = forceLineReturn(content);
+  content = forceLineReturn(content, this.normalizedCliWidth());
```

Comment:
Let's calculated the width value only once at the beginning of the function and reuse it elsewhere.